frontend/assets/javascripts/src/main.js

```diff
@@ -55,7 +56,8 @@ require([
     metrics,
     patterns,
     paidToPaid,
-    memstatus
+    memstatus,
+    campaignCode
```

Comment:
Nit: maybe this conceptually belongs in `src/modules/analytics/setup.js`, where the old omniture line used to be?